es-ES/readme.md

```diff
@@ -0,0 +1,703 @@
+
+AVA actualmente sólo transpilará los tests que usted le pida que ejecute. *No transpilará módulos que ```importen``` de fuera del test.* Si bien hay razones válidas para adoptar este enfoque, puede que no sea lo que esperas!
+
+Como solución simple, puede utilizar [Babel's require hook](https://babeljs.io/docs/usage/require/) con el fin de hacer transpilación sobre la marcha de los módulos que se importan posteriormente. Por que AVA soporta la sintaxis de módulo de ES2015, se puede utilizar para importar el hook requerida en sí:
+
+```js
+import test from 'ava';
+import 'babel-core/register';
+import foo from './foo'; // <-- foo puede ser escrito en ES2015!
+
+test('foo bar', t => {
+	t.same('baz', foo('bar'));
+});
+```
+
+[#111](https://github.com/sindresorhus/ava/issues/111) esta haciendo seguimiento de este tema como una mejora potencial.
+
+### Soporte a Promesas
+
+Si usted devuelve una promesa en el test no necesitará poner fin de manera explícita al test, ya que terminará cuando la promesa se resuelve.
+
+```js
+test(t => {
+	return somePromise().then(result => {
+		t.is(result, 'unicorn');
+	});
+});
+```
+
+### Soporte a Generadores
+
+AVA viene con soporte a [generadores](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*).
+
+```js
+test(function * (t) {
+	const value = yield generatorFn();
+	t.true(value);
+});
+```
+
+### Soporte a funciones async
+
+AVA viene con soporte a [funciones async](https://tc39.github.io/ecmascript-asyncawait/) *(async/await)*.
+
+```js
+test(async function (t) {
+	const value = await promiseFn();
+	t.true(value);
+});
+
+// async arrow function
+test(async t => {
+	const value = await promiseFn();
+	t.true(value);
+});
+```
+
+### Soporte a Observable
+
+AVA viene con soporte a [observables](https://github.com/zenparsing/es-observable).
+Si devuelve un observable a partir de un test, AVA automáticamente podrá consumirlo hasta su finalización antes de terminar el test.
+
+*Si no necesita usar "callback mode" o llamar a `t.end()`.*
+
+```js
+test(t => {
+	t.plan(3);
+	return Observable.of(1, 2, 3, 4, 5, 6)
+		.filter(n => {
+			// only even numbers
+			return n % 2 === 0;
+		})
+		.map(() => t.pass());
+});
+```
+
+### Soporte a Callback
+
+AVA es compatible con el uso de `t.end` como callback final al utilizar las API de callback de error-first y node-style. AVA considerará como error cualquier valor truthy pasado como primer argumento de `t.end`. Tenga en cuenta que `t.end` requiere "callback mode", que se puede activar mediante el uso de la cadena `test.cb`.
+
+```js
+test.cb(t => {
+	// t.end comprueba automáticamente para el error como primer argumento
+	fs.readFile('data.txt', t.end);
+});
+```
+
+### Salida TAP opcional
+
+AVA puede generar una salida TAP a través de la opción `--tap` para su uso con cualquier [TAP reporter](https://github.com/sindresorhus/awesome-tap#reporters).
+
+```
+$ ava --tap | tap-nyan
+```
+
+<img src="media/tap-output.png" width="398">
```

Comment:
To see correctly the image :
`<img src="media/tap-output.png" width="398">`
replace by : 
`<img src="https://github.com/sindresorhus/ava/blob/master/media/tap-output.png" width="398">`